pkg/sql/distsqlrun/backfiller_test.go

```diff
@@ -68,14 +70,41 @@ CREATE UNIQUE INDEX vidx ON t.test (v);
 		{Key: roachpb.Key("o"), EndKey: roachpb.Key("p")},
 		{Key: roachpb.Key("q"), EndKey: roachpb.Key("r")},
 	}
+
+	registry := server.JobRegistry().(*jobs.Registry)
+	tableDesc := sqlbase.GetTableDescriptor(kvDB, "t", "test")
+
 	if err := kvDB.Put(
-		context.TODO(),
+		ctx,
 		sqlbase.MakeDescMetadataKey(tableDesc.ID),
 		sqlbase.WrapDescriptor(tableDesc),
 	); err != nil {
 		t.Fatal(err)
 	}
 
+	mutationID := tableDesc.Mutations[0].MutationID
+	var jobID int64
+
+	if len(tableDesc.MutationJobs) > 0 {
+		for _, job := range tableDesc.MutationJobs {
+			if job.MutationID == mutationID {
+				jobID = job.JobID
+				break
+			}
+		}
+	}
+
+	job, err := registry.LoadJob(ctx, jobID)
+
+	if err != nil {
+		t.Fatal(errors.Wrapf(err, "can't find job %d", jobID))
+	}
+
+	details := job.Record.Details.(jobs.SchemaChangeDetails)
+
+	details.ResumeSpanList[0].ResumeSpans = resumeSpans
+	job.SetDetails(ctx, details)
+
```

Comment:
I think this can be moved into a function and used in this test and in the main code.